viz/cmd/authz.go

```diff
@@ -89,62 +91,55 @@ func NewCmdAuthz() *cobra.Command {
 				os.Exit(1)
 			}
 			for _, server := range servers {
-				sazs, err := k8s.ServerAuthorizationsForServer(cmd.Context(), k8sAPI, options.namespace, server)
+				requestParams := util.StatsSummaryRequestParams{
+					StatsBaseRequestParams: util.StatsBaseRequestParams{
+						TimeWindow:    options.timeWindow,
+						ResourceName:  server,
+						ResourceType:  k8s.Server,
+						Namespace:     options.namespace,
+						AllNamespaces: false,
+					},
+				}
+
+				req, err := util.BuildStatSummaryRequest(requestParams)
+				if err != nil {
+					return err
+				}
```

Comment:
i notice this returns an error, but the next line (`requestStatsFromAPI`) will exit immediately on error. is there a reason we return an error if building the request fails? do we handle that differently? it _might_ be worth a comment --- not a big deal though.

Reply:
The difference is that returning an error passes the error back to cobra which results in the cobra printing the error as well as the full usage info.  This is the behavior that we want when the user has given invalid input (e.g. causing us to fail to build the request) but isn't very helpful when there's an external error such as a failure from the API.  In that case, we exit directly to prevent cobra from printing usage info.